nautobot/dcim/views.py

```diff
@@ -2947,6 +2947,23 @@ class HardwareFamilyUIViewSet(NautobotUIViewSet):
     table_class = tables.HardwareFamilyTable
     lookup_field = "pk"
 
+    def get_extra_context(self, request, instance):
+        # Parent prefixes table
+        device_types = instance.device_types.select_related("manufacturer").annotate(
+            device_count=count_related(Device, "device_type")
+        )
```

Comment:
Probably should enforce user permissions on this queryset? A la:

```suggestion
        device_types = (
            DeviceType.objects.restrict(request.user, "view")
            .filter(hardware_family=instance)
            .select_related("manufacturer")
            .annotate(device_count=count_related(Device, "device_type"))
        )
```

Just looking at similar patterns in other views.